### PR TITLE
Various fixes

### PR DESCRIPTION
**Pull request by petergroth**
Changelist:

`train.py`
- Updated zero-shot column names to match ProteinGym
- Added try/except statement for loading checkpoints
- Changed `os.mkdir`to `os.makedirs(..., exist_ok=True)` to avoid parallel creations leading to crashes
 
`proteinnpt/baselines/model.py` 
- Comment out `ConvBertLayer` functionality (no longer supported by `transformers`)

`proteinnpt/proteinnpt/model.py`
- Comment out `ConvBertLayer` functionality (no longer supported by `transformers`)
- Fixed OHE dimension bug

`proteinnpt/utils.data_utils.py`
- Now reconstructs wild type sequence from mutated sequence and adds as argument to `cleanup_ids_assay_data()` to avoid errors

`proteinnpt/baselines/model_configs/OHE_DeepSequence_final.json`
- Fixed augmentation key to align with others and avoid errors

`proteinnpt/utils/data_utils.py`
- Remove wild type row from zero-shot dataframe to avoid errors (e.g. for DeepSequence)

### CODE REVIEW

File: scripts/train.py [hunk 1]
```diff
@@ -31,25 +31,25 @@ def setup_config_and_paths(args):
...
-            "Tranception_pred": "Tranception_L",
-            "DeepSequence_pred": "DeepSequence_ensemble"
+            "MSA_Transformer_pred": "esm_msa1b_t12_100M_UR50S_ensemble",
+            "ESM1v_pred": "Ensemble_ESM1v",
+            "ESM2_15B_pred": "esm2_t48_15B_UR50D",
+            "ESM2_3B_pred": "esm2_t36_3B_UR50D",
+            "ESM2_650M_pred": "esm2_t33_650M_UR50D",
+            "TranceptEVE_pred": "avg_score",
+            "Tranception_pred": "avg_score",
+            "DeepSequence_pred": "evol_indices_ensemble"
```
Comment (pascalnotin) [inline scripts/train.py:52]: Let's revert this change -- the prior names are the expected ones in the merged files / ProteinNPT data package.

File: proteinnpt/baselines/model.py [hunk 2]
```diff
@@ -66,7 +66,7 @@ def __init__(self, args, alphabet):
         if self.args.target_prediction_head == "AA_embeddings_mean_pooled":
             target_pred_input_dim = self.args.embed_dim
         elif self.args.target_prediction_head == "One_hot_encoding":
-            target_pred_input_dim = (self.args.seq_len + 1) * self.alphabet_size if args.target_prediction_model!="CNN" else self.alphabet_size    #Add one for the BOS token
+            target_pred_input_dim = (self.args.seq_len + 2) * self.alphabet_size if args.target_prediction_model!="CNN" else self.alphabet_size    #Add one for the BOS token
```
Comment (pascalnotin) [inline proteinnpt/baselines/model.py:69]: Why extending by 2 tokens here?

**Review by pascalnotin — COMMENTED** [review]
Thanks for the PR! Added a couple comments - let me know!

File: proteinnpt/baselines/model.py (hunk 2 above)
Comment (petergroth) [reply proteinnpt/baselines/model.py:69]: Default alphabet is `ESM-1b` (`train.py` line 210). This sets both `prepend_bos` and `append_eos` to True when instantiating `AugmentedPropertyPredictor`, so tokenized sequences are `seq_len + 2` long.

File: scripts/train.py (hunk 1 above)
Comment (petergroth) [reply scripts/train.py]: They're now reverted!

File: proteinnpt/baselines/model.py (hunk 2 above)
Comment (pascalnotin) [reply proteinnpt/baselines/model.py:69]: Got it - I must have been using the MSA Transformer Alphabet before refactoring the code for open release!

**Comment by pascalnotin** [comment]
Thanks for the updates @petergroth -- merging to main!
